import unittest


class TestCase_1_6_06(unittest.TestCase):
    def test_case_01(self):
        self.fail()

